lib/administrate/search.rb

```diff
@@ -31,6 +38,24 @@ def search_attributes
       end.keys
     end
 
-    attr_reader :resolver, :term
+    def search_scope(term)
+      if (term[-1, 1] == ':')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.